### DISCUSSION TITLE
How to unlink google with account?

### DISCUSSION THREAD

**Discussion by tusqasi** [monkeytypegame/monkeytype#2309] [Questions] [answered]
I maybe used google sign in at some point which was a mistake. I would like to change this to only sign in with my email.

Is there a way to do this already? I could not find it.

**Comment by Miodec** [comment, marked as answer]
Currently its not available through the UI but you can do it through the browser console by using the `unlinkGoogle()` command

**Reply by tusqasi** [reply]
i can i just slap some html button which calls this function and do a PR?

**Reply by Miodec** [reply]
Kinda dirty, it needs a few more checks and stuff. Ill be working on that today i think.
